public/javascripts/controllers/buyer-ctrl.js

```diff
@@ -168,9 +168,7 @@ angular.module('jasmic.controllers')
             /**
              * Fetches the units that user can select
              */
-             UnitsFactory.query({un_unit_class: "Weight"}, function(units){
-                    $scope.units = units;
-            });
+            $scope.units = UnitsFactory.query({});
```

Comment:
@nick22891 i believe this function should not have been removed. Can you explain its exclusion?